packages/documentation/pages/usage/components/filters.vue

```diff
@@ -120,12 +120,12 @@ export default defineComponent({
 	setup() {
 		const filters = ref<Kotti.Filters.Value>([
 			{
-				key: 'string-column',
+				key: 'stringColumn',
```

Comment:
@santiagoballadares how did this work before

isn't the key what you use to track changes in the `columns` prop

Reply:
The key is the identifier for each column, so yes. This change was only to have all values in camel case. All columns' keys were updated, so this shouldn't affect the filters at all.